<launch>
    <arg name="env_path" default="/home/ubuntu/S13P21A507/sim/simulator2/catkin_ws/.env" />
  
  <group ns="police_car_1">
    <node pkg="road_block" type="vehicle_registration.py" name="registration_p1" output="screen">
        <param name="car_name" value="POLICE_1" />
        <param name="vehicle_type" value="0" />
        <param name="env_path" value="$(arg env_path)" />
        <remap from="/Ego_topic" to="/Ego_topic" />
    </node>

    <node pkg="road_block" type="vehicle_locate_info.py" name="location_updater_p1" output="screen" respawn="true">
        <param name="env_path" value="$(arg env_path)" />
        <remap from="/Ego_topic" to="/Ego_topic" />
    </node>
    
    <node pkg="road_block" type="vehicle_stat_info.py" name="status_sender_p1" output="screen" respawn="true">
        <param name="env_path" value="$(arg env_path)" />
        <remap from="/Ego_topic" to="/Ego_topic" />
        <remap from="/CollisionData" to="/CollisionData" />
    </node>

    <node pkg="road_block" type="police_tracker_drive_backup.py" name="pursuit_controller_p1" output="screen">
        <remap from="/Ego_topic" to="/Ego_topic" />
        <remap from="/ctrl_cmd" to="/ctrl_cmd" />
        <remap from="/Ego_2/Ego_topic" to="/Ego_2/Ego_topic" />
    </node>
    
    <node pkg="road_block" type="chase_manager.py" name="chase_manager_p1" output="screen" respawn="true">
        <param name="env_path" value="$(arg env_path)" />
        <remap from="/Ego_topic" to="/Ego_topic" />
        <remap from="/ctrl_cmd" to="/ctrl_cmd" />
        <remap from="/fugitive_collision" to="/Ego_2/CollisionData" />
    </node>
  </group>
</launch>
